azure-pipelines.yml

```diff
@@ -292,3 +292,15 @@ stages:
               packagesToPush: '$(Pipeline.Workspace)/Module/$(Folder)/*.nupkg'
               nuGetFeedType: external
               publishFeedCredentials: 'PowerShell Gallery'
+              
+          - task: GithubRelease@0 
+            displayName: 'Create GitHub Release'      
+            inputs:
+              gitHubConnection: vsteam
+              repositoryName: DarqueWarrior/vsteam
+              tagSource: manual
+              tag: v$(PACKAGE_VERSION)      
+              addChangeLog: true
```

Comment:
@MichelZ how would that work? Which changelogs are being added here? Are those the once from git history?

Reply:
Yes.

From the docs:


> Add changelog | (Optional) If set to true,  a list of changes (commits and issues) between this and the last  published release will be generated and appended to release notes.

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/github-release?view=azure-devops